README.md

```diff
@@ -1,68 +1,873 @@
+      // session cookie:
+      console.log(res.user);
+      res.setCookieAndRedirect('https://acme.com/app');
+
+      // If the widget is configured for OIDC with a single responseType, the
+      // response will be the token.
+      // i.e. authParams.responseType = 'id_token':
+      console.log(res.claims);
+      signIn.tokenManager.add('my_id_token', res);
+
+      // If the widget is configured for OIDC with multiple responseTypes, the
+      // response will be an array of tokens:
+      // i.e. authParams.responseType = ['id_token', 'token']
+      signIn.tokenManager.add('my_id_token', res[0]);
+      signIn.tokenManager.add('my_access_token', res[1]);
+
+      return;
+    }
+
+  },
+
+  function error(err) {
+    // The widget will handle most types of errors - for example, if the user
+    // enters an invalid password or there are issues authenticating.
+    //
+    // This function is invoked with errors the widget cannot recover from:
+    // 1. Known errors: CONFIG_ERROR, UNSUPPORTED_BROWSER_ERROR, OAUTH_ERROR
+    // 2. Uncaught exceptions
+  }
+);
+```
+
+## session.get(callback)
+
+Gets the active session, or returns `{status:inactive}` on error or no active session.
+
+- `callback` - Function that is called with the session once the request has completed.
+
+```javascript
+signIn.session.get(function (res) {
+  // Session exists, show logged in state.
+  if (res.status === 'ACTIVE') {
+    // showApp()
+  }
+  // No session, or error retrieving the session. Render the Sign-In Widget.
+  else if (res.status === 'INACTIVE') {
+    signIn.renderEl(
+      {el: '#osw-container'},
+      function success(res) {
+        // showApp() if res.status === 'SUCCESS'
+      },
+      function error(err) {
+        // handleError(err)
+      }
+    );
+  }
+});
+```
+
+## session.refresh(callback)
+
+Refresh the current session by extending its lifetime. This can be used as a keep-alive operation.
+
+- `callback` - Function that is called after the refresh request has completed.
+
+```javascript
+signIn.session.refresh(function (res) {
+  if (res.status === 'ACTIVE') {
+    // The session now has an extended lifetime
+  }
+  else if (res.status === 'INACTIVE') {
+    // There is no current session, render the Sign-In Widget
+  }
+});
+```
+
+## session.close(callback)
+
+Signs the user out of their current Okta session.
+
+- `callback` - Function that is called once the session has been closed. If there is an error, it will be passed to the callback function.
+
+```javascript
+signIn.session.close(function (err) {
+  if (err) {
+    // The user has not been logged out, perform some error handling here.
+    return;
+  }
+  // The user is now logged out. Render the Sign-In Widget.
+});
+```
+
+## `OIDC` token.hasTokensInUrl()
+
+Synchronous method to check for access or id tokens in the url. This is used when `authParams.display = 'page'`. Returns `true` if there are tokens, and `false` if the redirect flow has not taken place yet.
```

Comment:
id tokens should be ID Tokens everywhere (we're following the spec conventions).